.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Kindly add node modules to your  .gitignore file, this will make sure in future you never accidentally add it when setting up linters

Reply:
@danmainah hello, please thanks for the review. I'm done now, can you kindly check it